src/compas/artists/shapeartist.py

```diff
@@ -34,7 +34,7 @@ class ShapeArtist(Artist):
 
     """
 
-    default_color = (1, 1, 1)
+    default_color = Color.from_hex('#0092D2').rgb
```

Comment:
Wouldn't it be better if the color attributes are actual Color instances than tuples?

Reply:
Not sure if this was required by other implementations tho

Reply:
yes, i have only done it for mesh for now as a prototype...

Reply:
well and a little test here and there :)